labs/splice.md

```diff
@@ -21,63 +21,93 @@ Daml is a platform built by Digital Asset designed for deploying blockchains and
+
+    * A Traffic Acquisition app, which allows users to purchase traffic across a Canton synchronizer, and configure automatic refills of a desired traffic balance as the user submits transactions via the synchronizer.
+
+Amulet behaviors are controlled by the underlying Daml models. These models can be used to create an on-chain instance of an amulet following the Amulet pattern, but the models do not control actual operations and governance of an on-chain amulet. Actual definition, configuration and governance of operating group-specific amulets takes place by setting configuration variables on these models via governance votes in a specific on-chain context.
+
+The Amulet models expose a large number of configuration variables to the Governance app (fifty-five variables in the current version). These allow synchronizer operating groups to use the Governance app to modify minting and burning behaviors, including but not limited to issuance curves, issuance round timing, fees, and rewards. This makes it possible to separate the governance of a particular implementation of the Amulet reference application from development of the Splice code base.
+
+We expect that contributors interested in Splice may contribute, for example, extensions to the APIs for the reference applications, including extensions to Amulet Wallet and the Payment Scan; extensions and enhancements to the Synchronizer Governance application; changes to Traffic Acquisition behavior and APIs, and enhanced UI features in the Wallet and the Name Service. These will be accepted based on the Hyperledger Foundation process for open source projects.
+
+Any group operating a Canton synchronizer may accept and implement new versions of the Splice reference applications at its own discretion. The amulet configuration used by any Canton synchronizer operating group can be made publicly available (visible) by that operating group.
 
 ## Architecture
-The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.<br>
+The following diagrams show how the Splice applications interact with the Daml platform from Digital Asset.
 
-![SV Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-1.jpg)
+![SV Node Architecture](./images/splice/Super%20Validator%20Architecture%20Diagram%20for%20Hyperledger%20Splice%20(2).jpg)
 
-![Validator Node Architecture](./images/Super%20Validator%20Architecture%20Diagram%20for%20Splice-2.jpg)
+![Validator Node Architecture](./images/splice/Super%20Validator%20Architecture%20Diagram%20for%20Hyperledger%20Splice%20(3).jpg)
```

Comment:
These images do not exist in the repo. Do they need to be added and the other images removed?

Reply:
fixed image url. sorry (again...)

Reply:
@tkuhrt fixed.